src/test/resources/org/padaiyal/utilities/commandline/CommandLineUtilityTest.properties

```diff
@@ -0,0 +1,17 @@
+# suppress inspection "UnusedProperty"
```

Comment:
Remove unnecessary 'suppress inspections'.

Reply:
It's needed for IntelliJ or any other IDE to not generate a warning.